script.js

```diff
@@ -14,6 +14,11 @@ const poems = [
     poem: "Roses are red \n Violets are blue \n Jerry Seinfeld is considering \n Making Bee Movie 2",
     image: "bee.jpeg",
   },
+  {
+    author: "James",
+    poem: "Roses are red \n This is a great poem for you to read",
```

Comment:
Consider rhyming. Looks good though ! 👍
(You don't have to do this, I'm just simulating real feedback)